.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
It would depend on your implementation 

I think we've got two ways forward. We can either refactor out the unused tokens or we can disable the checks locally. 

When you disable something in rubocop.yml it disables the cop from running entirely. In this case you're disabling a cop checking for unused keyword arguments. If an argument is unused it's best not to have it in the first place. Disabling it in the rubocop.yml will fix the problem at the surface but we would then never catch any new unused keyword arguments. 

In software - generally we should try to figure out what the root cause of a problem is and address that root problem. With this particular issue disabling all checks does is treating the symptom and not the cause.

Please either refactor the code so the keyword args are not used or disable the cop locally and I'll file a ticket for the refactor.